testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -140,30 +146,34 @@ def function(x):
 
         arguments = [tuple([i]) for i in np.arange(0,100)]
 
-        parallel_calculation = encore.utils.ParallelCalculation(function = function,
-                                                                ncores = 4,
-                                                                args = arguments)
+        parallel_calculation = encore.utils.ParallelCalculation(function=function,
+                                                                n_jobs=4,
+                                                                args=arguments)
         results = parallel_calculation.run()
 
         for i,r in enumerate(results):
             assert_equal(r[1], arguments[i][0]**2,
                 err_msg="Unexpeted results from ParallelCalculation")
 
-    def test_rmsd_matrix_with_superimposition(self):        
+    @dec.skipif(module_not_found('sklearn'),
+                "Test skipped because sklearn is not available.")
+    def test_rmsd_matrix_with_superimposition(self):
```

Comment:
scikit-learn distributes wheels. It is rather easy to install today on any platform (win/osx/unix).

I can also make a direct joblib dependency that is not a problem. My personal standpoint is just that an analysis may fail at any point if the recommended packages aren't installed. But we have to have a separate discussion about that anyway.